src/app/packages/open-layers-map/openlayers-map/openlayers-map.ts

```diff
@@ -278,26 +279,89 @@ export class OpenLayersMap extends IMap<OLMap> {
 		};
 	}
 
-	public setPosition(position: CaseMapPosition, view: View = this.mapObject.getView()): void {
-		const isProjectedPosition = view.getProjection().getCode() === position.projectedState.projection.code;
+	public setPosition(position: ICaseMapPosition, view: View = this.mapObject.getView()): void {
+		const viewProjection = view.getProjection();
+		const isProjectedPosition = viewProjection.getCode() === position.projectedState.projection.code;
 		const { extent, projectedState } = position;
 		const { center, rotation, zoom } = projectedState;
 		if (isProjectedPosition) {
 			view.setCenter(center);
 			view.setZoom(zoom);
-		} else {
+		} else if (position.resolutionData) {
+			const projectedCenter = proj.transform(position.resolutionData.center, 'EPSG:4326', viewProjection);
+			view.setCenter(projectedCenter);
+			this.setResolution(position.resolutionData);
+		}
+		else {
 			this.fitToExtent(extent, view);
 		}
 		this.setRotation(rotation, view);
 	}
 
-	public getPosition(): CaseMapPosition {
+	setResolution(resolutionData: ICaseResolutionData): void {
```

Comment:
`setResolution(resolutionData: ICaseResolutionData, view: View = this.mapObject.getView())`